website/docs/feature-flag-tutorials/node/implementing-feature-flags.md

```diff
@@ -0,0 +1,217 @@
+2. Now use the default credentials to log into the instance:
+
+```bash
+Username: admin
+Password: unleash4all
+```
+
+## Create a New Feature
+
+Create a new feature flag in your Unleash instance named `user-access`.
+
+![Create a new feature flag in Unleash](/img/gradual-rollout-userid-flag-create.png)
+
+## Integrating Unleash in Node.js app
+
+### Installation
+
+To get started with Node.js and Unleash, you need to install `unleash-client` package as a dependency.
+
+You can run the following commands in your terminal to do this:
+
+```bash
+npm install unleash-client
+```
+
+### Set up Environment Variables
+
+By default, the following values are setup in your local Unleash instance
+
+```bash
+# .env
+
+UNLEASH_API_URL="http://localhost:4242/api"
+UNLEASH_AUTHORIZATION_KEY="default:development.unleash-insecure-api-token"
+```
+
+### Initialize Unleash SDK
+
+To make feature flags available to our Node.js application, we will create a `unleash` helper file. This helper will initialize the Unleash SDK and provide access to feature flags throughout our application. We will do this by adding it to our `lib/unleash.js` file.
+
+```javascript
+// File: lib/unleash.ts
+
+require('dotenv').config()
+
+const { startUnleash } = require('unleash-client')
+
+module.exports = async function () {
+  return await startUnleash({
+    appName: 'user-access-levels',
+    url: process.env.UNLEASH_API_URL,
+    customHeaders: {
+      Authorization: process.env.UNLEASH_AUTHORIZATION_KEY,
+    },
+  })
+}
+```
+
+### Create a simple HTTP server with Node.js
+
+```javascript
+// File: index.js
+
+require('dotenv').config()
+
+// Specify the port and host to listen on
+const port = process.env.PORT || 3000
+const host = process.env.HOST || '0.0.0.0'
+
+const http = require('http')
+
+const server = http.createServer(async (req, res) => {
+    // Use req.url to conditionally send responses
+    // based on the url
+})
+
+// Start the server and listen on the specified host and port
+server.listen(port, host, () => {
+  console.log(`Listening on http://${host}:${port}`)
+})
+```
+
+### Use Unleash SDK to fetch the feature flag value
+
+Next, we will redirect the user upon entering their sign up information, and then use the `isEnabled` method to determine whether onboard them or not.
+
+```javascript
+// File: index.js
+
+// ..
+// Utilities Imports
+// ..
+
+const getUnleash = require('./lib/unleash')
+
+// Create an HTTP server
+const server = http.createServer(async (req, res) => {
+  if (req.url.includes('/dashboard/new')) {
+    
+    // ...
+    // Get form data
+    // This is just an example, but you might want to persist user data
+    // through cookies on the server and not rely on the request body
+    // to send it to the server for security reasons
+    // ...
+    
+    // Access the parsed form fields using the 'fields' object from formidable
+    const userId = fields.email[0]
+    
+    // Load Unleash SDK
+    const unleash = await getUnleash()
```

Comment:
This promotes an incorrect way to use Unleash. Unleash should be initialized outside of the context of the "request scope" of the application. 

Also highlighted in this comment: https://github.com/Unleash/unleash/pull/5246/files#r1385619950

Reply:
@ivarconr 

Thanks for teaching me that. I'm learning 😁

I've updated the docs and the git repo with that change.